templates/ovndbcluster/bin/setup.sh

```diff
@@ -79,20 +79,21 @@ set "$@" --ovn-${DB_TYPE}-logfile=/dev/null
 # don't log to file (we already log to console)
 $@ ${OPTS} run_${DB_TYPE}_ovsdb -- -vfile:off &
 
+CTLCMD="ovn-${DB_TYPE}ctl --no-leader-only"
 if [[ "$(hostname)" == "{{ .SERVICE_NAME }}-0" ]]; then
     # The command will wait until the daemon is connected and the DB is available
     # All following ctl invocation will use the local DB replica in the daemon
-    export OVN_${DB_TYPE^^}_DAEMON=$(ovn-${DB_TYPE}ctl --pidfile --detach)
+    export OVN_${DB_TYPE^^}_DAEMON=$(${CTLCMD} --pidfile --detach)
 
 {{- if .TLS }}
-    ovn-${DB_TYPE}ctl --no-leader-only set-ssl {{.OVNDB_KEY_PATH}} {{.OVNDB_CERT_PATH}} {{.OVNDB_CACERT_PATH}}
-    ovn-${DB_TYPE}ctl --no-leader-only set-connection ${DB_SCHEME}:${DB_PORT}:0.0.0.0
+    ${CTLCMD} set-ssl {{.OVNDB_KEY_PATH}} {{.OVNDB_CERT_PATH}} {{.OVNDB_CACERT_PATH}}
```

Comment:
Is this required? The ctl commands should only connect to the local running daemon so the option has no effect. It makes sense to me only for the daemon process

Reply:
I am not sure if this is needed; but I don't want to touch it (it's moved from elsewhere). I asked elsewhere why we need these since we also pass --ssl related arguments through CLI. We can follow up with cleanup if needed.